android/build.gradle

```diff
@@ -5,36 +5,33 @@ def safeExtGet(prop, fallback) {
 buildscript {
     repositories {
         jcenter()
+        google()
     }
+
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:3.1.4'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', 26)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.1')
+    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
 
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
         targetSdkVersion safeExtGet('targetSdkVersion', 26)
         versionCode 1
         versionName "1.0"
-        ndk {
-            abiFilters "armeabi-v7a", "x86"
-        }
+    }
+    lintOptions {
+        abortOnError false
     }
 }
 
 repositories {
-    maven {
-        // As RN is not updated in jcenter anymore, we'll have to grab it from npm
-        url "../node_modules/react-native/android"
-    }
```

Comment:
This is necessary since RN comes from npm

Reply:
Yeah, sorry about this. Reverted!